tests/iDibo_tests.py

```diff
@@ -1,11 +1,24 @@
 from nose.tools import *
-import iDibo
 
-def setup():
-    print "SETUP!"
+import unittest
+from gluon.globals import Request
 
-def teardown():
-    print "TEAR DOWN!"
+db = test_db
 
-def test_basic():
-    print "I RAN!"
\ No newline at end of file
+execfile("applications/iDibo/controllers/default.py", globals())
+db(db.auth_user.id > 0).delete()
+db.commit()
+
+
+class TestRegister(unittest.TestCase):
+    def setup(self):
+        request = Request()
+
+    def test_register_title(self):
+        self.controller.register()
+        my_title = self.controller.register.response.title
+        self.assertEqual(my_title, 'iDibo Registration')
+
+
+if __name__ == '__main__':
+    unittest.main()
```

Comment:
maybe d execfile actually worked though.
but i had tried lot of tweaking earlier to access various keys but it keeps saying no value, although it seems it sees the fact that the returned type is actually an sqlform